### PR TITLE
Fixed Typo and grammar  saas.md

### PR DESCRIPTION
**Pull request by Sanyam-2026**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — APPROVED** [review]
@Sanyam-2026 Looks good, passing to a second review.

**Comment by SSwiniarski** [comment]
> > @Sanyam-2026 Looks good, passing to a second review.
> 
> @SSwiniarski Waiting for second review since last 3 days

@Sanyam-2026 have patience. There are only a few maintainers and a high volume of PRs to review.

**Comment by Sanyam-2026** [comment]
> > > @Sanyam-2026 Looks good, passing to a second review.
> > 
> > 
> > @SSwiniarski Waiting for second review since last 3 days
> 
> @Sanyam-2026 have patience. There are only a few maintainers and a high volume of PRs to review.
yes I can understand but can it be faster @SSwiniarski can you tag the second reviewer so it can be merged soon

**Review by KTom101 — APPROVED** [review]

**Pull request merged**
